src/installer.ts

```diff
@@ -8,13 +8,13 @@ import * as exec from '@actions/exec';
 
 let osPlat: string = os.platform();
 
-export async function getBuildx(version: string): Promise<string> {
+export async function getBuildx(version: string, dockerConfigHome: string): Promise<string> {
```

Comment:
I've opted for inversion of control, here, in order to ease the test case I've written.

FWIT, I've tries [this approach](https://github.com/vuejs/vue-test-utils/issues/193#issuecomment-607769018) but it only works when my test case is isolated (I'm not fluent in the JS/TS ecosystem).